vic/drivers/image/include/vic_driver_image.h

```diff
@@ -187,6 +203,7 @@ void vic_start(void);
 void vic_store(void);
 void vic_write(void);
 char will_it_snow(double *t, double t_offset, double max_snow_temp,
-                  double *prcp, size_t n);
+                  double *prcp,
+                  size_t n);
```

Comment:
out of curiosity, what version of `uncrustify` are you using?  @bartnijssen and I using `0.60`.